### ISSUE TITLE
TextEditor: Requires components.js

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#3315]
Based on this forum post: https://forum.primefaces.org/viewtopic.php?f=3&t=52597

TextEditor does not work if it is the only component on the page with this error:

```javascript
'$' is undefined
core.js.xhtml (2,420)

Unable to get property 'DeferredWidget' of undefined or null reference
texteditor.js.xhtml (19,1)

'$' is undefined
prime.xhtml (4,168)
```

**Comment by tandraschko** [comment]
hm? DefferedWidget should be in core.js, not components.js

**Comment by melloware** [comment]
Let me double check again. That is what I thought too.

**Comment by melloware** [comment]
You are right this was already fixed:  https://github.com/primefaces/primefaces/issues/2345

My mistake was the user was using PF 6.1.4 which I thought it was fixed in since I fixed it last May but it was put into the 6.2 release not the 6.1.X branch!
